### PR TITLE
adds phone_number field to user registration

### PR DESCRIPTION
**Pull request by Eerian**

### CODE REVIEW

File: frontend/src/components/AddProductForm/AddProductForm.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-        Name: <input type="name" name="name" required/>
-        Price: <input type="number" name="price" required/>
+        Name: <input type="name" name="name" required />
+        Price: <input type="number" name="price" required />
         <input type="submit" value="Submit" />
       </form>
     </>
-  )
-}
-export default AddProductForm
```
Comment (onionsalot) [inline frontend/src/components/AddProductForm/AddProductForm.js:27]: Any reason this component was modified? I don't see any diffs besides lints

Comment (Eerian) [reply frontend/src/components/AddProductForm/AddProductForm.js:27]: I think I miss-clicked on it and when I saved it and my linter just added semicolons and spaces

Comment (onionsalot) [reply frontend/src/components/AddProductForm/AddProductForm.js:27]: SGTM. You can restore it from main using
`git fetch
git restore -s origin/main-- frontend/src/components/AddProductForm/AddProductForm.js`

Commit the changes after restoring it and the file should disappear from your diffs.

Comment (angel-zh) [reply frontend/src/components/AddProductForm/AddProductForm.js:27]: Perhaps we can edit the linter settings to not add semicolons, so our files can be consistent across

Comment (onionsalot) [reply frontend/src/components/AddProductForm/AddProductForm.js:27]: I've added some settings that will help to keep everything a bit more consistent when using prettier in the README. @Eerian I suggest you install those then run the linter on the JS files you changed to keep the design language the same-ish

Comment (Eerian) [reply frontend/src/components/AddProductForm/AddProductForm.js]: ok this should be resolved now

**Review by onionsalot — APPROVED** [review]

**Pull request merged**